### PR TITLE
Add ip validation

### PR DESCRIPTION
**Pull request by amiraabouhadid**
### Changes
checks if IP exists in all farms 

### Related Issues

#2726

### CODE REVIEW

File: packages/playground/src/dashboard/components/add_ip.vue [hunk 1]
```diff
@@ -201,6 +212,12 @@ export default {
           message: "IP is not public.",
         };
       }
+      const ips = await gqlClient.publicIps({ ip: true }, { where: { ip_eq: publicIP.value } });
+      if (ips.length > 0) {
+        return {
+          message: "IP exists",
+        };
+      }
```
Comment (zaelgohary) [inline packages/playground/src/dashboard/components/add_ip.vue:220]: We use the same function implementation twice. Can we extract it outside the block and call it in both places?

File: packages/playground/src/dashboard/components/add_ip.vue [hunk 2]
```diff
@@ -146,12 +149,19 @@ export default {
...
+    async function ipcheck() {
       if (PrivateIp(publicIP.value.split("/")[0])) {
         return {
           message: "IP is not public",
         };
       }
+
+      if (await IpExistsCheck()) {
+        return {
+          message: "IP exists",
```
Comment (zaelgohary) [inline packages/playground/src/dashboard/components/add_ip.vue:161]:
```suggestion
          message: "IP exists.",
```

File: packages/playground/src/dashboard/components/add_ip.vue [hunk 3]
```diff
@@ -201,6 +214,11 @@ export default {
           message: "IP is not public.",
         };
       }
+      if (await IpExistsCheck()) {
+        return {
+          message: "IP exists",
```
Comment (zaelgohary) [inline packages/playground/src/dashboard/components/add_ip.vue:219]:
```suggestion
          message: "IP exists.",
```

**Review by zaelgohary — COMMENTED** [review]
Good job, Amira! just small comment.

**Review by maayarosama — APPROVED** [review]

**Review by MohamedElmdary — APPROVED** [review]

**Pull request merged**